test/test_create_entities.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+import time
+
+from test.unified_format import UnifiedSpecTestMixinV1
+
+
+class TestCreateEntities(unittest.TestCase):
+
+    def setUp(self):
+        self.scenario_runner = UnifiedSpecTestMixinV1()
```

Comment:
I think we can remove the setUp method here since there's only one test.

Reply:
Sure but each test can create its own UnifiedSpecTestMixinV1. Normally we'd reach for setUp() only once there's more boilerplate.

Reply:
That is true there isn't much boilerplate. Done.